src/Components/Book.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Book = (props) => {
+  // eslint-disable-next-line react/prop-types
+  const { bookProps } = props;
+  return (
+    <div>
+      {/* eslint-disable-next-line react/prop-types */}
+      <p>{bookProps.title}</p>
+      {/* eslint-disable-next-line react/prop-types */}
+      <p>{bookProps.author}</p>
+      <button type="button">Delete</button>
+    </div>
```

Comment:
- [ ] [OPTIONAL] Kindly notice that it is better to use destructuring in `props` like this.

```javascript
const { title, author } = props;
  return (
    <div>
      {/* eslint-disable-next-line react/prop-types */}
      <p>{title}</p>
      {/* eslint-disable-next-line react/prop-types */}
      <p>{author}</p>
      <button type="button">Delete</button>
    </div>
  );
```